src/app/api/auth/[...nextauth]/route.ts

```
@@ -0,0 +1,82 @@
+      },
+      async authorize(credentials) {
+        if (!credentials) return null;
+        
+        const { email, password } = credentials as {
+          email: string;
+          password: string;
+        };
+
+        try {
+          // Apply rate limiting
+          const { success, limit, remaining, reset } = await rateLimit(email);
```

ESLint

[error] 'limit' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)